core/src/main/java/org/elasticsearch/search/aggregations/AggregatorBuilder.java

```diff
@@ -62,6 +62,47 @@ public AggregatorBuilder(String name, Type type) {
     }
 
     /**
+     * Read from a stream.
+     */
+    protected AggregatorBuilder(StreamInput in, Type type) throws IOException {
+        name = in.readString();
+        this.type = type;
+        factoriesBuilder = AggregatorFactories.Builder.PROTOTYPE.readFrom(in);
+        metaData = in.readMap();
+    }
+
+    protected boolean usesNewStyleSerialization() {
```

Comment:
Presumably this should be removed when we have migrated all the builders to the new serialisation? If so, can we add a NORELEASE comment so we don't forget?